README.md

```diff
@@ -37,22 +37,28 @@ Then run `cargo build`.
 ## Usage
 
 ```rust
```

Comment:
Correctness of this example is not checked via CI, right? I find the risk of eventually being outdated not worth the benefit of including an example on the main README.md. How about only linking to `examples/readme.rs`?